### PR TITLE
Improve HTML rendering of table length information

### PR DESCRIPTION
**Pull request by saurvs**
https://github.com/astropy/astropy/issues/5685

Update the following tests:
- [x] TestPprint
- [x] TestRow

### CODE REVIEW

**Comment by saurvs** [comment]
@pllim I'm curious, should `_base_repr_` for `Table` remove the `<`/`>` for table length when `html=False` as well?

**Comment by pllim** [comment]
>  should `_base_repr_` for `Table` remove the `</>` for table length when `html=False`?

It is a question for @astrofrog or @taldcroft . :sweat_smile:

**Comment by saurvs** [comment]
My bad!

File: astropy/table/table.py [hunk 1]
```diff
@@ -809,12 +809,13 @@ def _base_repr_(self, html=False, descr_vals=None, max_width=None,
...
-
+        descr = ' '.join(descr_vals)
         if html:
             from ..utils.xml.writer import xml_escape
-            descr = xml_escape(descr)
-
+            descr = '<i>{0}</i>\n'.format(xml_escape(descr))
+        else:
+            descr = '<{0}>\n'.format(descr)
+
```
Comment (bsipocz) [inline astropy/table/table.py]: Please remove the extra whitespace from this line (as it triggers the style checker to fail).

**Comment by saurvs** [comment]
@bsipocz can you take a look?

**Comment by bsipocz** [comment]
@saurvs - style tests now pass, but the PR still needs a review from the module maintainers.

**Comment by mhvk** [comment]
I'll leave this one to @taldcroft, who has thought more about html rendering.

**Comment by bsipocz** [comment]
@taldcroft @astrofrog - Can you have a look at this, is this addresses the original issue?

**Comment by astrofrog** [comment]
Oops the bot updated the wrong comment (the warning got overwritten by a changelog entry message). Should be fixed now.

**Comment by astrofrog** [comment]
Preview:

screen shot 2017-09-27 at 14 18 48

**Pull request merged**